src/Umbraco.Web.Common/ApplicationBuilder/IUmbracoEndpointBuilderContext.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.AspNetCore.Routing;
+
+namespace Umbraco.Cms.Web.Common.ApplicationBuilder
+{
+
+    /// <summary>
+    /// A builder to allow encapsulating the enabled routing features in Umbraco
```

Comment:
Looks like this comment needs updating, as the interface is for a context now rather than a builder.